### PR TITLE
Enables CTF after round end.

### PR DESCRIPTION
**Pull request by Nirnael**
Enables CTF after round end.

:cl: Nirnael
tweak: CTF can now be played after round ends.
/:cl:

### CODE REVIEW

File: code/modules/awaymissions/capture_the_flag.dm [hunk 1]
```diff
@@ -133,7 +133,7 @@
 /obj/machinery/capture_the_flag/attack_ghost(mob/user)
 	if(ctf_enabled == FALSE)
 		return
-	if(ticker.current_state != GAME_STATE_PLAYING)
+	if(ticker.current_state >= GAME_STATE_PLAYING)
 		return
```
Comment (X-TheDark) [inline code/modules/awaymissions/capture_the_flag.dm:137]:
``` c
#define GAME_STATE_PLAYING  3
#define GAME_STATE_FINISHED 4
```

Erm...not quite. Need `<=`

**Comment by george99g** [comment]
I know this is a single character change, but...

``` c
//ticker.current_state values
#define GAME_STATE_STARTUP        0
#define GAME_STATE_PREGAME        1
#define GAME_STATE_SETTING_UP    2
#define GAME_STATE_PLAYING        3
#define GAME_STATE_FINISHED        4
```

Shouldn't this be `<=` instead, since `>=` will make it not work while the game is playing and after it already ends, due to the return.

**Comment by Nirnael** [comment]
Yes sorry, my bad.

**Comment by X-TheDark** [comment]
Wonder if CTF should enable automatically when the round end gets delayed (due to tickets).

**Comment by Nirnael** [comment]
Hmm sounds like a good idea, i'll do it in a bit.

**Comment by Nirnael** [comment]
Actually, gonna leave it at that for now, if anyone wants to do it, do it, if not I'll do it when I'm done with my stuff. I'm not coding anything longer than a few lines yet.

**Pull request merged**